FirebaseAdmin/FirebaseAdmin/Auth/Jwt/FirebaseTokenFactory.cs

```diff
@@ -173,14 +176,75 @@ internal static FirebaseTokenFactory Create(FirebaseApp app, string tenantId = n
                 header, payload, this.Signer, cancellationToken).ConfigureAwait(false);
         }
 
+        internal async Task<string> CreateCustomTokenAppIdAsync(
```

Comment:
I feel like this should be isolated into the `FirebaseAdmin.AppCheck` namespace. Why did you put app check specific code into the `FirebaseAdmin.Auth` namespace? I don't see anything `AppCheck` related in the `auth` directory in the JS SDK either.

Reply:
I have updated code and the issue has been resolved.